test/clj/chronograph/handlers/organization_test.clj

```diff
@@ -93,3 +93,49 @@
         (is (s/valid? :organizations/organization
                       (:body response))
             "Slug length can be exactly 256 characters at most.")))))
+
+
+(deftest find-one-organization
+  (testing "when fetching an organization"
+    (let [user (factories/create-user)
+          other-user (factories/create-user)
+          slug "bar"
+          slug-that-doesnt-exist (str (java.util.UUID/randomUUID))
+          organization (:body (create-organization
```

Comment:
Could use the domain function to create the organization, or make a factory function for it. I think the handler may be a bit clunky for this purpose.

Reply:
We have a factory function to create organization, which wraps over the corresponding domain fn. However our factory returns the owner-id for some reason. Hm, I'll see what to do about this...

Reply:
Bah, I misread the factory fn. It returns the created organization as it should. :facepalm: 

Using the factory fn now, which in turn uses the domain fn to create the organization.